orm/solid-start/README.md

```diff
@@ -0,0 +1,160 @@
+# SolidStart Example
+
+This example shows how to implement a simple web app using [SolidStart](https://start.solidjs.com) and [Prisma ORM](https://www.prisma.io/docs).
+
+## Getting started
+
+### 1. Download the example and navigate to the project directory
+
+Download this example:
+
+```
+npx try-prisma@latest --template orm/solid-start
+```
+
+Then navigate to the project directory
+
+```
+cd solid-start
+```
+
+<details><summary><strong>Alternative:</strong> Clone the entire repo</summary>
+
+Clone this repository:
+
+```
+git clone git@github.com:prisma/prisma-examples.git --depth=1
+```
+
+Install npm dependencies:
+
+```
+cd prisma-examples/orm/solid-start
+npm install
+```
+
+</details>
+
+#### [Optional] Switch database to Prisma Postgres
+
+This example uses a local SQLite database by default. If you want to use to [Prisma Postgres](https://prisma.io/postgres), follow these instructions (otherwise, skip to the next step):
+
+1. Set up a new Prisma Postgres instance in the Prisma Data Platform [Console](https://console.prisma.io) and copy the database connection URL.
+2. Update the `datasource` block to use `postgresql` as the `provider` and paste the database connection URL as the value for `url`:
+
+   ```prisma
+   datasource db {
+     provider = "postgresql"
+     url      = "prisma+postgres://accelerate.prisma-data.net/?api_key=ey...."
+   }
+   ```
+
+   > **Note**: In production environments, we recommend that you set your connection URL via an [environment variable](https://www.prisma.io/docs/orm/more/development-environment/environment-variables/managing-env-files-and-setting-variables), e.g. using a `.env` file.
+
+3. Install the Prisma Accelerate extension:
+   ```
+   npm install @prisma/extension-accelerate
+   ```
+4. Add the Accelerate extension to the `PrismaClient` instance:
+
+   ```diff
+   + import { withAccelerate } from "@prisma/extension-accelerate"
+
+   + const prisma = new PrismaClient().$extends(withAccelerate())
+   ```
+
+That's it, your project is now configured to use Prisma Postgres!
+
+### 2. Generate Prisma Client
+
+Run the following command to generate the Prisma Client. This is what you will be using to interact with your database.
+
+```
+npx prisma generate
+```
+
+### 3. Start the SolidStart server
+
+```
+npm run dev
+```
+
+The server is now running at http://localhost:3000. You can now view your todo list!
+
+## Switch to another database (e.g. PostgreSQL, MySQL, SQL Server, MongoDB)
```

Comment:
I see why you did this and we may have done it with other examples, but I think a better solution here would be to just reference our documentation on how someone can use a different db engine/provider.